client/src/containers/Workspaces/Workspaces.js

```diff
@@ -1,56 +1,51 @@
-import React, { useEffect, useState } from 'react';
+import React, { useState } from 'react';
 import { Link } from 'react-router-dom';
+import useFetch from '../../hooks/useFetch';
+import { useAppContext } from '../App/AppContext';
 
 import Container from '../../components/Container/Container';
 import InputField from '../../components/InputField/InputField';
 import SidebarList from '../../components/SidebarList/SidebarList';
 import Spinner from '../../components/Spinner/Spinner';
 import AddWorkspaceModal from './AddWorkspace';
 
-
 const Workspaces = () => {
-  const [ownedWS, setOwnedWS] = useState([]);
-  const [joinedWS, setJoinedWS] = useState([]);
-  const [isLoading, setLoading] = useState(true);
   const [modalVisibility, setModalVisibility] = useState(false);
 
+  const user = useAppContext();
+
+  const fetchOwnedWorkspaces = useFetch('post', '/workspace/all', {
```

Comment:
Why is the dummy/hardcoded data here? You could always create a gist and share separately, rather than committing it directly into the code.